vars/systemTests.groovy

```diff
@@ -49,36 +50,70 @@ void call(Map config = [:]) {
         ]
 
     echo "Starting System-Tests with parameters: ${buildParameters}"
+    parallel(
+        dv: {
```

Comment:
`vars/systemTests.groovy` - is a helper function to trigger downstream pipeline. It can be called from anywhere.
This change modifies it so it triggers two downstream pipelines in `parallel`. I can see two points to consider:
1. There is a risk this won't work, and won't be displayed nicely in Jenkins. There is a Jankins limit that parallel cannot be nested. So if the `systemTests` function is already called from within parallel, then `systemTests` function will throw an exception.
2. Would it be better to add an argument: `useCapsule: true/false`, and in Jenkinsfile where it is used just call it twice?